build.gradle.kts

```diff
@@ -45,36 +46,46 @@ buildscript {
 @Suppress("RemoveRedundantQualifierName") // Cannot use imports here.
 plugins {
     `java-library`
+    kotlin("jvm") version "1.4.30"
     idea
-    id("com.google.protobuf") version io.spine.gradle.internal.Deps.build.protobuf.gradlePluginVersion
-    id("net.ltgt.errorprone") version io.spine.gradle.internal.Deps.build.errorProne.gradlePluginVersion
+    @Suppress("RemoveRedundantQualifierName") // Cannot use imports here.
+    io.spine.gradle.internal.Deps.build.apply {
```

Comment:
I'm surprised this works. `plugins {}` DSL in very limited. This is possibly because `apply` is an `inline` method. Good to know!